src/routers/message.ts

```diff
@@ -424,3 +439,23 @@ addHandler(
     res.json(message);
   }
 );
+
+addHandler(
+  messageRouter,
+  "get",
+  addApiV1Prefix("/legal-messages/:id"),
+  (req, res) => {
+    // tslint:disable-next-line:no-commented-code
+    // if (configResponse.getLegalMessageResponseCode !== 200) {
+    //   res.sendStatus(configResponse.getLegalMessageResponseCode);
+    //   return;
+    // }
+
+    // retrieve the messageIndex from id
+    const message = messagesWithContent.find(item => item.id === req.params.id);
+    if (message === undefined) {
+      res.json(getProblemJson(404, "message not found"));
+    }
+    res.json(message);
```

Comment:
Should we also check that the request message is a legal message?
```suggestion
    if (LegalMessageWithContent.decode(message).isLeft()) {
      res.json(getProblemJson(400, "bad parameter"));
      return;
    }
    res.json(message);
```

Reply:
It's a good observation, though I don't know how the API would respond in this case.
Maybe @fabriziopapi can answer this question.

Reply:
We already know what this API returns in case of `200`
https://github.com/pagopa/io-backend/blob/df2c4bb5a62df3ce2f9254c04ca56bae18032032/api_backend.yaml#L280
I believe we should ensure that from the given id we are actually returning a legal message detail

Reply:
I think we are overthinking it: if the point of the dev-server is to mimick the actual API, we must know how it would behave if a client asked for an ID which is a message but not a _legal_ one.
Otherwise, 404 is totally fine in my opinion.

Reply:
Updated in https://github.com/pagopa/io-dev-api-server/pull/107/commits/4373e149d34a0d2d4b754b8b0e1e370ea9b25777 as per our chat. Now returning a meaningful message which will help us during development.